storage/client/file/src/test/resources/session-records/clearMetadataFromDirClient.json

```diff
@@ -1,42 +1,42 @@
 {
   "networkCallRecords" : [ {
     "Method" : "PUT",
-    "Uri" : "https://sima.file.core.windows.net/dirsharename/directory863334?restype=directory",
+    "Uri" : "https://sima.file.core.windows.net/dirsharename/directory06962b?restype=directory",
```

Comment:
We shouldn't have checked-in tests that reference a specific dev's account.  Can you create a tracking work item to move these Uri's to a team account versus an individual account?
/cc @weshaggard

Reply:
Yes we should be setting up these resources more generally in our test resources subscription. @danieljurek I believe you have some docs talking about that correct?

Reply:
This would be an easy change. Will do in this PR.

Reply:
Yes. The short version is that **test resources belong in the test resources subscription** .. Details here: https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki?pagePath=%2FData%20Plane%20Quality%20Program%2FTesting%20Guidelines&pageId=51&wikiVersion=GBwikiMaster&anchor=best-practices

Reply:
We have switched the azstoragesdkaccount team account under Azure SDK Test Resource subscription already.